vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.cc

```diff
@@ -142,14 +142,19 @@ void Promotion::OnGetAllPromotions(
 
   if (list.size() > 0) {
     for (auto & item : list) {
-      auto it = promotions.find(item->id);
-      if (it != promotions.end()) {
-        if (promotions.at(item->id)->status !=
-            ledger::PromotionStatus::ACTIVE) {
-          item->status = promotions.at(item->id)->status;
-        }
+      if (item->legacy_claimed) {
+        item->status = ledger::PromotionStatus::CLAIMED;
+        ClaimTokens(item->Clone(), [](const ledger::Result _){});
```

Comment:
there is a `_`  symbol after `Result`.

Reply:
that is needed

Reply:
Per:
https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions
`A parameter name may be omitted only if the parameter is not used in the function's definition.`
The `_` is considered as a special symbol. We usually use it as trailing part in class/struct members names.